src/lib/ui/next/customNextPage.js

```diff
@@ -6,17 +6,13 @@ import GradientBG from '../components/GradientBG.js';
 import styles from '../styles/Home.module.css';
 import heroMinaLogo from '../public/assets/hero-mina-logo.svg';
 import arrowRightSmall from '../public/assets/arrow-right-small.svg';
-import './reactCOIServiceWorker';
 import {fetchAccount, Mina, PublicKey} from "o1js";
 import {Add} from "../../contracts";
 
-
 // We've already deployed the Add contract on testnet at this address
 // https://minascan.io/devnet/account/B62qnTDEeYtBHBePA4yhCt4TCgDtA4L2CGvK7PirbJyX4pKH8bmtWe5
 const zkAppAddress = "B62qnTDEeYtBHBePA4yhCt4TCgDtA4L2CGvK7PirbJyX4pKH8bmtWe5";
 
-import './reactCOIServiceWorker';
```

Comment:
Ah thank you for catching this!

Reply:
It it was not me, our e2e CI did :)
You can also run e2e tests locally.